test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/OnRemoteControlSettings/001_SUCCESS_in_case_AUTO_ALLOW_seq_1.lua

```diff
@@ -0,0 +1,63 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/10
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/resource_allocation.md
+-- Item: Use Case 1: Main Flow
+--
+-- Requirement summary:
+-- [SDL_RC] Resource allocation based on access mode
+--
+-- Description: TRS: OnRemoteControlSettings, #1
+-- In case:
+-- 1) SDL received OnRemoteControlSettings notification from HMI with allowed:true
+-- 2) and "accessMode" = "AUTO_ALLOW" or without "accessMode" parameter at all
+-- 3) and RC_module on HMI is alreay in control by RC-application
+-- SDL must:
+-- 1) provide access to RC_module for the second RC_application in HMILevel FULL after it sends control RPC
+-- (either SetInteriorVehicleData or ButtonPress) for the same RC_module without asking a driver
+-- 2) process the request from the second RC_application
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/commonRCmodules')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+--modules array does not contain "RADIO" because "RADIO" module has read only parameters
+local modules = { "CLIMATE", "AUDIO", "LIGHT", "HMI_SETTINGS" }
```

Comment:
@GetmanetsIrina Suggest to use only modules related to the proposal

Reply:
@GetmanetsIrina Please ignore my previous comment since proposal also describes __new__ (related to proposal) parameters